src/index.ts

```diff
@@ -744,26 +755,67 @@ class Datastore extends DatastoreRequest {
    *  The location prefix of an App Engine project ID.
    *  Often this value is 's~', but may also be 'e~', or other location prefixes
    *  currently unknown.
-   * @returns {string} base64 endocded urlsafe key.
+   * @param {function} callback The callback function.
+   * @param {?error} callback.err An error returned while making this request
+   * @param {string} callback.urlSafeKey base64 endocded urlsafe key.
    *
    * @example
    * const {Datastore} = require('@google-cloud/datastore');
    * const datastore = new Datastore();
    * const key = datastore.key(['Company', 'Google']);
    *
-   * datastore.keyToLegacyUrlsafe(key) //ag9ncmFzcy1jbHVtcC00NzlyEwsSB0NvbXBhbnkiBkdvb2dsZQw
+   * datastore.keyToLegacyUrlSafe(key, (err, urlSafeKey) => {
+   *   if (err) {
+   *     // Error handling omitted.
+   *   }
+   *   console.log(urlSafeKey);
+   * })
    *
-   * @example
-   * <caption>Create a complete url safe key using location prefix </caption>
+   * //-
+   * // Create a complete url safe key using location prefix
+   * //-
    * const {Datastore} = require('@google-cloud/datastore');
    * const datastore = new Datastore();
    * const key = datastore.key(['Task', 123]);
    * const locationPrefix = 's~';
    *
-   * datastore.keyToLegacyUrlsafe(key, locationPrefix) //ahFzfmdyYXNzLWNsdW1wLTQ3OXIKCxIEVGFzaxh7DA
+   * datastore.keyToLegacyUrlSafe(key, locationPrefix, (err, urlSafeKey) => {
+   *   if (err) {
+   *     // Error handling omitted.
+   *   }
+   *   console.log(urlSafeKey);
+   *
+   * //-
+   * // If the callback is omitted, we'll return a Promise.
+   * //-
+   * const {Datastore} = require('@google-cloud/datastore');
+   * const datastore = new Datastore();
+   * const key = datastore.key(['Task', 123]);
+   * const locationPrefix = 's~';
+   *
+   * const urlSafeKey = await datastore.keyToLegacyUrlSafe(key, locationPrefix);
```

Comment:
To be consistent with our other examples in the repo, let's show this as a promise:

```js
datastore.keyToLegacyUrlSafe(key).then((data) => {
  const urlSafeKey = data[0];
});
```